src/media/js/addon/containers/submit.js

```diff
@@ -58,8 +60,7 @@ export class AddonSubmit extends React.Component {
   render() {
     return (
       <section>
-        <h1>Submitting a Firefox OS Add-on</h1>
-
```

Comment:
might've kept a line-break for logical separation, the form section is pretty big